wgengine/wgcfg/nmcfg/nmcfg.go

```diff
@@ -60,6 +61,7 @@ func WGCfg(nm *netmap.NetworkMap, logf logger.Logf, flags netmap.WGConfigFlags,
 		Peers:      make([]wgcfg.Peer, 0, len(nm.Peers)),
 	}
 
+	skippedUnselected := new(bytes.Buffer)
```

Comment:
what about the line 108 ones?

Reply:
They aren't getting rate limited, so I didn't bother with them. If you'd like I can give them all the same treatment.

Reply:
Did the rest.